modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney/request_controller.rb

```diff
@@ -94,6 +93,54 @@ def create
 
         private
 
+        def validate_decide_params!(proc_id:, decision:)
+          if proc_id.blank?
+            raise ::Common::Exceptions::ParameterMissing.new('procId',
+                                                             detail: 'procId is required')
```

Comment:
Probably more of a nit.  Given this would be an issue with the request I'm not sure if we need to update this to meet the LH standards (could also be a follow up ticket if an addition to the LH errors is required)?

`If an error is in the request, a source field should point to it.`

Technically _should_ means it is not required I believe, just a recommendation, but still think it might be worth verifying if we at least want a follow up ticket to do that.

Reply:
Letʼs address the errors in another task since we use this pattern elsewhere in this controller.